controllers/permits/applications/special-uses/validate.js

```diff
@@ -304,7 +308,45 @@ const validateInput = function (route, inputPost){
 
 };
 
+const validateFile = function (uploadFile){
+
+	let returnError = '';
+
+	const regex = '(rtf|doc|docx|pdf)';
+
+	const tempErrors = [];
+
+	tempErrors.push(uploadFile.filetype + ' file errors:');
+
+	if (uploadFile.ext && !uploadFile.ext.toLowerCase().match(regex)){
+		tempErrors.push('invalid extension,');
+	}
+	else if (fileMimes.indexOf(uploadFile.mimetype) < 0){
+		tempErrors.push('invalid mimetype,');		
+	}
+	if (uploadFile.size === 0){
+		tempErrors.push('size of 0 bytes,');
+	}
+	else {
+		const fileSizeInMegabytes = uploadFile.size / 1000000.0;
+		if (fileSizeInMegabytes > 100){
+			tempErrors.push('size is greater than 100 MB,');	
+		}
+	}
+
+	if (tempErrors.length > 1){
+
+		returnError = util.concatErrors(tempErrors);
+
+		returnError = returnError.trim().replace(/.$/,'.'); // eslint-disable-line comma-spacing
```

Comment:
No need to disable comma-spacing here, just add the space. That shouldn't break anything.